source/authentication.rst

```diff
@@ -351,12 +346,28 @@ Alternatively, edit the configuration file:
    setParameter:
      ldapConnectionPoolSizePerHost: 5
 
+.. rubric:: Support for multiple LDAP servers
+
+As of version 3.6.21-11.0, you can specify multiple LDAP servers for failover. |PSMDB| sends authentication requests to the first server defined in the list. When this server is down or unavailable, it sends requests to the next server  and so on. Note that |PSMDB| keeps sending requests to this server even after the unavailable server recovers.
+
+Specify the LDAP servers as a comma-separated list in the format ``<host>:<port>`` for the `--ldapServers <https://docs.mongodb.com/manual/reference/program/mongoldap/#cmdoption-mongoldap-ldapservers>`_ option. 
+
+.. code-block:: yaml
+
+   security:
+     authorization: "enabled"
+     ldap:
+       servers: "ldap1.example.net", "ldap2.example.net"
```

Comment:
Is it a correct definition? Or should values be included in a single quotation marks?

Reply:
Nope, the correct one is either 
servers: "ldap1.example.net,ldap2.example.net" 
OR
servers: ldap1.example.net,ldap2.example.net